themes/theme-without-style-book-theme/src/css/_clay_variables.scss

```diff
@@ -0,0 +1,3 @@
+@import "variables/colors";
+@import "variables/button";
+@import "variables/typography";
```

Comment:
By convention, typography file should be named as `_type.scss`.

Reply:
Done